packages/create-gatsby/src/components/select.js

```diff
@@ -32,30 +38,33 @@ export class SelectInput extends Select {
       return undefined
     }
 
-    let styles = this.styles
-    let focused = this.index === i
-    let style = focused ? styles.primary : val => val
-    let ele = await this.resolve(val[focused ? `on` : `off`] || val, this.state)
+    const styles = this.styles
+    const focused = this.index === i
+    const style = focused ? styles.primary : val => val
+    const ele = await this.resolve(
+      val[focused ? `on` : `off`] || val,
+      this.state
+    )
     return focused ? style(ele) : ` `.repeat(ele.length)
   }
 
   async render() {
-    let { submitted, size } = this.state
+    const { submitted, size } = this.state
```

Comment:
my formatter shooted 🤦🏻 . I can revert if necessary

Reply:
No, That's fine. It's code that I lifted from enquirer